[2019-10-04 12:12:56,006] {taskinstance.py:620} INFO - Dependencies all met for <TaskInstance: capstone_project_dag.Load_visa_types_dim_table 2019-10-04T17:11:49.652690+00:00 [queued]>
[2019-10-04 12:12:56,030] {taskinstance.py:620} INFO - Dependencies all met for <TaskInstance: capstone_project_dag.Load_visa_types_dim_table 2019-10-04T17:11:49.652690+00:00 [queued]>
[2019-10-04 12:12:56,030] {taskinstance.py:838} INFO - 
--------------------------------------------------------------------------------
[2019-10-04 12:12:56,030] {taskinstance.py:839} INFO - Starting attempt 1 of 4
[2019-10-04 12:12:56,030] {taskinstance.py:840} INFO - 
--------------------------------------------------------------------------------
[2019-10-04 12:12:56,102] {taskinstance.py:859} INFO - Executing <Task(LoadDimFileData): Load_visa_types_dim_table> on 2019-10-04T17:11:49.652690+00:00
[2019-10-04 12:12:56,102] {base_task_runner.py:133} INFO - Running: ['airflow', 'run', 'capstone_project_dag', 'Load_visa_types_dim_table', '2019-10-04T17:11:49.652690+00:00', '--job_id', '498', '--pool', 'default_pool', '--raw', '-sd', 'DAGS_FOLDER/project_dag.py', '--cfg_path', '/var/folders/dr/9yvjlnm11xv25k2gknf2n3z00000gn/T/tmpbqdjgw24']
[2019-10-04 12:12:58,296] {base_task_runner.py:115} INFO - Job 498: Subtask Load_visa_types_dim_table /usr/local/lib/python3.7/site-packages/airflow/configuration.py:627: DeprecationWarning: You have two airflow.cfg files: /Users/victorialemay/airflow/airflow.cfg and /Users/victorialemay/Courses/Udacity/Data Engineering Nanodegree/capstone/project-archive/airflow/airflow.cfg. Airflow used to look at ~/airflow/airflow.cfg, even when AIRFLOW_HOME was set to a different value. Airflow will now only read /Users/victorialemay/Courses/Udacity/Data Engineering Nanodegree/capstone/project-archive/airflow/airflow.cfg, and you should remove the other file
[2019-10-04 12:12:58,297] {base_task_runner.py:115} INFO - Job 498: Subtask Load_visa_types_dim_table   category=DeprecationWarning,
[2019-10-04 12:12:58,606] {base_task_runner.py:115} INFO - Job 498: Subtask Load_visa_types_dim_table [2019-10-04 12:12:58,606] {settings.py:213} INFO - settings.configure_orm(): Using pool settings. pool_size=5, max_overflow=10, pool_recycle=1800, pid=66781
[2019-10-04 12:12:59,432] {base_task_runner.py:115} INFO - Job 498: Subtask Load_visa_types_dim_table [2019-10-04 12:12:59,431] {__init__.py:51} INFO - Using executor LocalExecutor
[2019-10-04 12:13:00,685] {base_task_runner.py:115} INFO - Job 498: Subtask Load_visa_types_dim_table [2019-10-04 12:13:00,685] {dagbag.py:90} INFO - Filling up the DagBag from /Users/victorialemay/Courses/Udacity/Data Engineering Nanodegree/capstone/project-archive/airflow/dags/project_dag.py
[2019-10-04 12:13:00,904] {base_task_runner.py:115} INFO - Job 498: Subtask Load_visa_types_dim_table [2019-10-04 12:13:00,904] {cli.py:516} INFO - Running <TaskInstance: capstone_project_dag.Load_visa_types_dim_table 2019-10-04T17:11:49.652690+00:00 [running]> on host 1.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.ip6.arpa
[2019-10-04 12:13:00,959] {load_dim_file_data.py:35} INFO - Using data file visa_types.json from S3 bucket project-capstone
[2019-10-04 12:13:01,063] {logging_mixin.py:95} INFO - [[34m2019-10-04 12:13:01,063[0m] {[34mbase_hook.py:[0m84} INFO[0m - Using connection to: [1mid: redshift. Host: redshift-cluster-1.c4i0rfzboubh.us-west-2.redshift.amazonaws.com, Port: 5439, Schema: dev, Login: awsuser, Password: XXXXXXXX, extra: {}[0m[0m
[2019-10-04 12:13:01,806] {load_dim_file_data.py:44} INFO - (0,)
[2019-10-04 12:13:01,807] {load_dim_file_data.py:47} INFO - Inserting into destination Redshift table visa_types
[2019-10-04 12:13:01,820] {logging_mixin.py:95} INFO - [[34m2019-10-04 12:13:01,820[0m] {[34mbase_hook.py:[0m84} INFO[0m - Using connection to: [1mid: redshift. Host: redshift-cluster-1.c4i0rfzboubh.us-west-2.redshift.amazonaws.com, Port: 5439, Schema: dev, Login: awsuser, Password: XXXXXXXX, extra: {}[0m[0m
[2019-10-04 12:13:02,383] {logging_mixin.py:95} INFO - [[34m2019-10-04 12:13:02,383[0m] {[34mdbapi_hook.py:[0m171} INFO[0m - 
        COPY visa_types 
        FROM 's3://project-capstone/visa_types.json'
        ACCESS_KEY_ID 'AKIASZIXFGD4JFXXFSEI'
        SECRET_ACCESS_KEY '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'
        JSON 'auto'
    [0m
[2019-10-04 12:13:05,062] {taskinstance.py:1051} ERROR - Load into table 'visa_types' failed.  Check 'stl_load_errors' system table for details.
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 926, in _run_raw_task
    result = task_copy.execute(context=context)
  File "/Users/victorialemay/Courses/Udacity/Data Engineering Nanodegree/capstone/project-archive/airflow/plugins/operators/load_dim_file_data.py", line 57, in execute
    redshift.run(formatted_sql)
  File "/usr/local/lib/python3.7/site-packages/airflow/hooks/dbapi_hook.py", line 172, in run
    cur.execute(s)
psycopg2.errors.InternalError_: Load into table 'visa_types' failed.  Check 'stl_load_errors' system table for details.

[2019-10-04 12:13:05,066] {taskinstance.py:1074} INFO - Marking task as UP_FOR_RETRY
[2019-10-04 12:13:05,105] {base_task_runner.py:115} INFO - Job 498: Subtask Load_visa_types_dim_table Traceback (most recent call last):
[2019-10-04 12:13:05,105] {base_task_runner.py:115} INFO - Job 498: Subtask Load_visa_types_dim_table   File "/usr/local/bin/airflow", line 32, in <module>
[2019-10-04 12:13:05,105] {base_task_runner.py:115} INFO - Job 498: Subtask Load_visa_types_dim_table     args.func(args)
[2019-10-04 12:13:05,105] {base_task_runner.py:115} INFO - Job 498: Subtask Load_visa_types_dim_table   File "/usr/local/lib/python3.7/site-packages/airflow/utils/cli.py", line 74, in wrapper
[2019-10-04 12:13:05,105] {base_task_runner.py:115} INFO - Job 498: Subtask Load_visa_types_dim_table     return f(*args, **kwargs)
[2019-10-04 12:13:05,106] {base_task_runner.py:115} INFO - Job 498: Subtask Load_visa_types_dim_table   File "/usr/local/lib/python3.7/site-packages/airflow/bin/cli.py", line 522, in run
[2019-10-04 12:13:05,106] {base_task_runner.py:115} INFO - Job 498: Subtask Load_visa_types_dim_table     _run(args, dag, ti)
[2019-10-04 12:13:05,106] {base_task_runner.py:115} INFO - Job 498: Subtask Load_visa_types_dim_table   File "/usr/local/lib/python3.7/site-packages/airflow/bin/cli.py", line 440, in _run
[2019-10-04 12:13:05,106] {base_task_runner.py:115} INFO - Job 498: Subtask Load_visa_types_dim_table     pool=args.pool,
[2019-10-04 12:13:05,106] {base_task_runner.py:115} INFO - Job 498: Subtask Load_visa_types_dim_table   File "/usr/local/lib/python3.7/site-packages/airflow/utils/db.py", line 74, in wrapper
[2019-10-04 12:13:05,106] {base_task_runner.py:115} INFO - Job 498: Subtask Load_visa_types_dim_table     return func(*args, **kwargs)
[2019-10-04 12:13:05,106] {base_task_runner.py:115} INFO - Job 498: Subtask Load_visa_types_dim_table   File "/usr/local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 926, in _run_raw_task
[2019-10-04 12:13:05,106] {base_task_runner.py:115} INFO - Job 498: Subtask Load_visa_types_dim_table     result = task_copy.execute(context=context)
[2019-10-04 12:13:05,107] {base_task_runner.py:115} INFO - Job 498: Subtask Load_visa_types_dim_table   File "/Users/victorialemay/Courses/Udacity/Data Engineering Nanodegree/capstone/project-archive/airflow/plugins/operators/load_dim_file_data.py", line 57, in execute
[2019-10-04 12:13:05,107] {base_task_runner.py:115} INFO - Job 498: Subtask Load_visa_types_dim_table     redshift.run(formatted_sql)
[2019-10-04 12:13:05,107] {base_task_runner.py:115} INFO - Job 498: Subtask Load_visa_types_dim_table   File "/usr/local/lib/python3.7/site-packages/airflow/hooks/dbapi_hook.py", line 172, in run
[2019-10-04 12:13:05,107] {base_task_runner.py:115} INFO - Job 498: Subtask Load_visa_types_dim_table     cur.execute(s)
[2019-10-04 12:13:05,107] {base_task_runner.py:115} INFO - Job 498: Subtask Load_visa_types_dim_table psycopg2.errors.InternalError_: Load into table 'visa_types' failed.  Check 'stl_load_errors' system table for details.
[2019-10-04 12:13:05,107] {base_task_runner.py:115} INFO - Job 498: Subtask Load_visa_types_dim_table 
[2019-10-04 12:13:05,929] {logging_mixin.py:95} INFO - [[34m2019-10-04 12:13:05,927[0m] {[34mlocal_task_job.py:[0m105} INFO[0m - Task exited with return code 1[0m
